src/MongoDB/Cursor.c

```diff
@@ -72,17 +72,32 @@ PHP_METHOD(Cursor, setTypeMap)
 
 static int php_phongo_cursor_to_array_apply(zend_object_iterator *iter, void *puser TSRMLS_DC) /* {{{ */
 {
-	zval **data, *return_value = (zval*)puser;
+#if PHP_VERSION_ID >= 70000
+	zval *data, *return_value = (zval*)puser;
```

Comment:
Since the return_value is zval* in both cases, I think its better to split this line into two.
The original is also inconsistent with how we do this everywhere else in the code so I think that will only be a good thing.

I realize it will need an extra if version check for the data assignment below, but I think that will have to be ok